airflow/providers/amazon/aws/auth_manager/aws_auth_manager.py

```diff
@@ -197,8 +206,14 @@ def is_authorized_view(
         )
 
     def is_authorized_custom_view(
-        self, *, method: ResourceMethod | str, resource_name: str, user: BaseUser | None = None
+        self,
+        *,
+        method: ResourceMethod | str,
+        resource_name: str,
+        user: BaseUser | None = None,
+        **kwargs,
     ):
+        # TODO: remove this if block when min_airflow_version is set to higher than 2.9.0
```

Comment:
Should that comment be in the `__init__` ?

Reply:
I need a coffee a think

Reply:
You're right